src/rules/selector-combinator-space-after/index.js

```diff
@@ -44,7 +44,7 @@ export function selectorCombinatorSpaceChecker({ locationChecker, root, result,
       styleSearch({
         source: selector,
         target: _.toArray(nonSpaceCombinators),
-        functionArguments: "skip",
+        parentheticals: "skip",
```

Comment:
What about `a:not(li + a)`, in which a real combinator is inside parentheses? Any way we can check that?